tests/features/authentication.feature

```diff
@@ -0,0 +1,10 @@
+@api
+Feature: Authentication
+  In order to allow personalized actions to be taken
```

Comment:
I guess that was just a missing test... nothing related with body classes? You confirm?